src/help/guides/getting-started.md

```diff
@@ -23,8 +27,7 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
 :::
 
 ### AltStore instructions for Mac
-1. Copy "**AltServer.app**" to your *Applications* folder.
-1. Launch **AltServer** from your menu bar (there should be an icon).
```

Comment:
I may not understood this step. 
I think we need to use AltServer from the menue bar in the `AltStore Mail plug-in for Mac` section but I do not see what do we need to do here.

Reply:
I think you can replace it with "Copy AltServer.app to your Applications folder and launch it."

Most mac users already know how to launch applications that are in the Applications folder since they are either available in Launchpad or Spotlight

Reply:
Okay. Wouldn't "Move" be better than "copy"?

Reply:
Either would be fine

Reply:
Done